tests/lib/BackgroundJob/DummyJobList.php

```diff
@@ -46,6 +46,10 @@ public function add($job, $argument = null): void {
 		}
 	}
 
+	public function scheduleAfter(string $job, int $runAfter, $argument = null): void {
+		$this->add($job, $argument);
```

Comment:
```suggestion
		$this->add($job, $argument, $runAfter);
```

Would have expected $runAfter as third argument here, no?